### ISSUE TITLE
Screen readers do not announce information on the chart.

### ISSUE THREAD

**Issue by Vignesh-SF3580** [syncfusion/maui-toolkit#247]
### Description

Please refer to the following .NET MAUI link
https://github.com/dotnet/maui/issues/30801

### Version with bug

1.0.4

### Is this a regression from previous behavior?

Not sure, haven't tested other versions

### Last Known Working Version

1.0.3

### Affected platforms

Windows, Android, macOS, iOS

### Relevant log output

```shell

```
